doc/source/operating.rst

```diff
@@ -263,4 +263,52 @@ your file system). The procedure to restore is no more complicated than initial
 deployment: restore your index and storage backups, reinstall Gnocchi if
 necessary, and restart it.
 
+How to clear Gnocchi data
+=========================
+
+If you ever want to start fresh or needs to clean Gnocchi data, this can be
+easily done. You need to clean the storage data, the incoming data and the
+index data.
+
+Once that is done, if you want to re-initialize Gnocchi, you need to call
+`gnocchi-upgrade` so it re-initialize the different drivers.
+
+Index data
+----------
+
+Both MySQL and PostgreSQL drivers uses a single database. Delete the database.
+If you want to install Gnocchi again, create back that database with the same
+name before calling `gnocchi-upgrade`.
+
+Incoming data
```

Comment:
(the same) Measure storage

Reply:
Well the thing is that the conf section is `[incoming]` etc, so that might confusing.